### PR TITLE
doc/rados/ops: edit monitoring-osd-pg.rst (2 of x)

### PR DESCRIPTION
**Pull request by zdover23**
Line-edit monitoring-osd-pg.rst (2 of x).

https://tracker.ceph.com/issues/58485

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rados/operations/monitoring-osd-pg.rst [hunk 1]
```diff
@@ -300,187 +301,188 @@ to the placement group.
...
 
 
 Clean 
 -----
 
-When a placement group is in the ``clean`` state, the primary OSD and the
-replica OSDs have successfully peered and there are no stray replicas for the
-placement group. Ceph replicated all objects in the placement group the correct 
-number of times.
+When a PG is in the ``clean`` state, the primary OSD and the replica OSDs have
```
Comment (anthonyeleven) [inline doc/rados/operations/monitoring-osd-pg.rst:335]: state, all OSDs holding its data and metadata have

File: doc/rados/operations/monitoring-osd-pg.rst [hunk 2]
```diff
@@ -300,187 +301,188 @@ to the placement group.
...
 OSDs that Ceph created the replica objects successfully. 
 
-The reason a placement group can be ``active+degraded`` is that an OSD may be
-``active`` even though it doesn't hold all of the objects yet. If an OSD goes
-``down``, Ceph marks each placement group assigned to the OSD as ``degraded``.
-The OSDs must peer again when the OSD comes back online. However, a client can
-still write a new object to a ``degraded`` placement group if it is ``active``.
+The reason that a PG can be ``active+degraded`` is that an OSD can be ``active``
+even if it doesn't yet hold all of the objects. If an OSD goes ``down``, Ceph
+marks each PG assigned to the OSD as ``degraded``. The OSDs must peer again when
```
Comment (anthonyeleven) [inline doc/rados/operations/monitoring-osd-pg.rst:351]: PGs must peer again

File: doc/rados/operations/monitoring-osd-pg.rst [hunk 3]
```diff
@@ -300,187 +301,188 @@ to the placement group.
...
 to another OSD. The time between being marked ``down`` and being marked ``out``
-is controlled by ``mon_osd_down_out_interval``, which is set to ``600`` seconds
+is determined by ``mon_osd_down_out_interval``, which is set to ``600`` seconds
 by default.
 
-A placement group can also be ``degraded``, because Ceph cannot find one or more
-objects that Ceph thinks should be in the placement group. While you cannot
-read or write to unfound objects, you can still access all of the other objects
-in the ``degraded`` placement group.
+A PG can also be in the ``degraded`` condition because there are one or more
```
Comment (anthonyeleven) [inline doc/rados/operations/monitoring-osd-pg.rst:361]: condition/state

**Review by anthonyeleven — APPROVED** [review]

File: doc/rados/operations/monitoring-osd-pg.rst [hunk 4]
```diff
@@ -284,12 +284,13 @@ The following subsections describe the common pg states in detail.
...
 
-When you create a pool, it will create the number of placement groups you
-specified. Ceph will echo ``creating`` when it is creating one or more
-placement groups. Once they are created, the OSDs that are part of a placement
-group's Acting Set will peer. Once peering is complete, the placement group
-status should be ``active+clean``, which means a Ceph client can begin writing
-to the placement group.
+PGs are created when you create a pool: the command that creates a pool
+specifies the total number of PGs for that pool, and when the pool is created
+all of those PGs are created as well. Ceph will echo ``creating`` while it is
```
[the remark this answers is not in the document]
Comment (zdover23) [reply doc/rados/operations/monitoring-osd-pg.rst:289]: unfinished business

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/50945 - Reef backport
https://github.com/ceph/ceph/pull/50946 - Quincy backport
https://github.com/ceph/ceph/pull/50947 - Pacific backport

**Pull request merged**